### ISSUE TITLE
Funky GUI

### ISSUE THREAD

**Issue by DFNCTSC** [tonikelope/megabasterd#151]
@tonikelope  I'm really getting tired of this UI. It's not organized or designed properly at all. I would REALLY like to help you with my UI/UX experience and help organize the layout of megabasterd's interface. If you'd allow me to, please reply and let me know so we can exchange some contact details and beautify the most useful MEGA downloader there is. :D

**Comment by savefromgoogle** [comment]
I agree, but I think the first sentence isn't really helping your cause. You'd have more success if you'd have left that one out.

What I'd _really_ like to see in MegaBasterd, at least, is buttons for "Move to top/bottom". Would be great to specify this when one is adding links, too. It would make the program that much better.

**Comment by savefromgoogle** [comment]
@tonikelope Wow, that was fast! Thanks for implementing those buttons this quickly!

**Comment by Hunam6** [comment]
It has nothing to do with that, @DFNCTSC was talking about the fact that it could make a design and organization suggestion that @tonikelope could then try to implement in the application.

**Comment by savefromgoogle** [comment]
@Hunam6 Yes, I have understood the request from OP. I was merely giving thanks for the [commit faf74fa / release 6.84](https://github.com/tonikelope/megabasterd/commit/faf74fa529709141bbf887c5c5c486987d114e64) which was made within a day after my initial comment and directly implemented what I suggested / asked about in the second sentence. I think there's a correlation between these events. I didn't have a better place to leave my thanks, so I did so here.
